apps/webstore/src/app/pages/auth/checkout/checkout.component.ts

```diff
@@ -372,7 +397,9 @@ export class CheckoutComponent implements OnInit {
           this.isLoading = false;
           console.log('Created order and got a payment link', paymentLink);
           this.localStorageService.removeItem(LocalStorageVars.discount);
+          this.localStorageService.removeItem(LocalStorageVars.plantedTrees);
           window.open(paymentLink.url, '_blank');
+          this.router.navigate(['/payment/success']);
```

Comment:
SHould navigate directly to the quickpay link instead of opening it another tab

```suggestion
          this.router.navigate(['paymentLink.url']);
```